test/options/rule-delimiter-scss/rule-delimiter.expected.scss

```diff
@@ -0,0 +1,19 @@
+.foo {
+  @import border-radius(5px);
+  background: red;
+
+.bar {
+
```

Comment:
The option should add new lines only `between` rulesets, not when there is nothing before.